app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -400,5 +402,11 @@ class SettingsFragment : PreferenceFragmentCompat(), AccountObserver {
             preferenceFirefoxAccountAuthError?.isVisible = false
             accountPreferenceCategory?.isVisible = false
         }
+
+        // if FxA has been turned off on the server
+        if (context?.isInExperiment(asFeatureFxADisabled)!!) {
+            val preferenceSignIn = findPreference<Preference>(context!!.getPreferenceKey(pref_key_sign_in))
+            preferenceSignIn?.isVisible = false
+        }
```

Comment:
IF we want we can do an `else` here to show some "Temporary out of service" graphic

Reply:
thinking about this, we cannot do this for MVP because we cannot add new strings